### PR TITLE
Mt performance pack test

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

**Comment by IkaDemisto** [comment]
Related to: https://github.com/demisto/etc/issues/31389

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 1]
```diff
@@ -1151,9 +1156,22 @@ def get_from_version_and_to_version_bounderies(all_modified_files_paths: set, id
...
+        pack_metadata_path = os.path.join(tools.pack_name_to_path(metadata), PACKS_PACK_META_FILE_NAME)
+        pack_metadata = get_pack_metadata(pack_metadata_path)
+        from_version = pack_metadata.get('serverMinVersion')
+        to_version = pack_metadata.get('serverMaxVersion')
+        if from_version:
+            min_from_version = min(min_from_version, LooseVersion(from_version))
+            max_from_version = max(max_from_version, LooseVersion(from_version))
+        if to_version:
+            max_to_version = max(max_to_version, LooseVersion(to_version))
+
```
Comment (hod-alpert) [inline Tests/scripts/collect_tests_and_content_packs.py:1174]: Lets make this a different method so we can add unittests for it

Comment (hod-alpert) [reply Tests/scripts/collect_tests_and_content_packs.py:1174]: Especially the case where we have the beta word in the value

Comment (yaakovi) [reply Tests/scripts/collect_tests_and_content_packs.py:1174]: 1) no reason to split to another function, I will add unit tests to it in the new PR.
2) `beta` version is handled by `LooseVersion` but I will add a test for it too.

**Pull request closed without merging**